apps/wallet-mobile/src/Catalyst/QrCode.tsx

```diff
@@ -119,36 +129,42 @@ const messages = defineMessages({
   },
 })
 
-const styles = StyleSheet.create({
-  safeAreaView: {
-    flex: 1,
-    backgroundColor: 'white',
-  },
-  alertBox: {
-    padding: 16,
-    backgroundColor: COLORS.BACKGROUND_LIGHT_RED,
-    borderRadius: 8,
-  },
-  contentContainer: {
-    paddingHorizontal: 16,
-  },
-  note: {
-    color: '#242838',
-    fontWeight: 'bold',
-  },
-  qrCodeBackground: {
-    borderRadius: 8,
-    padding: 16,
-    backgroundColor: '#F0F3F5',
-    alignSelf: 'center',
-  },
-  secretCodeBox: {
-    backgroundColor: '#F0F3F5',
-    borderRadius: 8,
-    padding: 16,
-    flexDirection: 'row',
-  },
-})
+const useStyles = () => {
+  const {theme} = useTheme()
+  const {color, padding} = theme
+  const styles = StyleSheet.create({
+    safeAreaView: {
+      flex: 1,
+      backgroundColor: color.gray.min,
+    },
+    alertBox: {
+      ...padding['l'],
+      backgroundColor: 'rgba(255, 16, 81, 0.06)',
```

Comment:
Is this color not in theme?

Reply:
yes, did not add this case as was waiting to see the product decision related to it 🤔